sphinx/workshop/reference.rst

```diff
@@ -14,6 +14,71 @@ Part 2
 
 You can download the helper library :download:`here <../../lib/basic_python3.py>`.
 
+Lists
+^^^^^^^^^^^^^^^^^^^^^
+
+Lists are structures that contain multiple values, and can contain numbers, strings and even other lists. 
+Like strings, you can add lists together, and assign them to variables. 
+You can also access an element of a list by using the *index* of the element. The first element has index 0. 
+
+.. code-block:: python
+
+    >>> lst = [1,2,3]
+    >>> lst2 = ["a","b","c"]
+    >>> lst + lst2
+    [1, 2, 3, 'a', 'b', 'c']
+    >>> newlst = lst + lst2
+    >>> newlst
+    [1, 2, 3, 'a', 'b', 'c']
+    >>> newlst[0]
+    1
+
+For loops
+^^^^^^^^^^^^^^^^^^^^^
+
+For loops enable us to loop over the elements in a list or string. 
+
+.. code-block:: python
+
+    >>> lst = [1,2,3,4]
+    >>> x = 0
+    >>> for i in lst:
+    	x += i
+    >>> x
+    10
+    >>> string = "hello"
+    >>> newstring = ""
+    >>> for char in string:
```

Comment:
Likewise, iteration over a string is not helpful to the beginner. This becomes problematic for the first time beginner who thinks of strings and numbers as building blocks---how is it that basic building blocks can be 'deconstructed'?